### PR TITLE
Port documentation #4276 to the docs repository

### PR DESCRIPTION
**Pull request by settermjd**
This change ports https://github.com/owncloud/documentation/pull/4276 to the docs repository.

💁‍♂️ **Do Not Merge**

### CODE REVIEW

**Comment by mmattel** [comment]
PLEASE DON NOT MERGE YET !
Issue: the code may work, but the term `shipped` is not used as it is literally thought.
Please see the original documentation issue and its links for more details.
Core has to make it somehow available what is marked shipped with the tar file.

**Comment by settermjd** [comment]
Thanks for spotting that, @mmattel.

**Comment by settermjd** [comment]
@voroyam, can you provide further assistance/input please?

**Comment by mmattel** [comment]
I can add some news. There is a PR open in core which tries to fix the issue of identifying apps beeing part of core. As long there is no core solution, no further steps can be taken.
https://github.com/owncloud/core/pull/34563 (getAppsInTar)

**Comment by voroyam** [comment]
We explained "shipped" in the upgrade documentation.

If this is the only reason not to merge this then we can merge from my pov.(after we resolve the conflicts)

**Comment by mmattel** [comment]
@voroyam just fyi, if you carefully read the original PR as referenced on top and the referenced PR, you may recognize the table and further documented details which show how the shipped flag is currently used and does not fulfill the requirements for the fundamental idea this PR stands for. Merging this PR leads to the situation for expecting something which just cant be fulfilled as not implemented and the system stability suffer from that. That was the reason for setting and keeping the wait label. You may ask @PVince81 to get more clarity as he is involved in the referenced core PR.

**Comment by settermjd** [comment]
@PVince81, can this PR be merged, as-is?

**Comment by mmattel** [comment]
@settermjd sorry we can´t. `occ app:list --shipped=false` is not doing that what we think it should do.
Please see https://github.com/owncloud/documentation/pull/4276 and https://github.com/owncloud/core/issues/32048
The text part moving the apps needs a revision and the script needs to be removed.
A working solution to identify which apps are definitely part of the shipped tar is to untar the source file and to look into that /apps folder. All apps residing in the installed /apps folder and not beeing part of the shipped tar /apps folder can be moved to apps-external. Incase, all double present apps residing in an existing /apps-external and /apps can safely be removed from the /apps-external folder.

**Comment by voroyam** [comment]
@mmattel can you tell me what is the worst thing that can happen?

Have you been using that tag and have run in to issues?

We have been using the shipped tag in the appliance upgrade for 2-3 versions and it works. It deactivates the apps that are not shipped and so there is no pesky “deactivate these apps and run occ upgrade again” message during upgrade.

**Comment by mmattel** [comment]
You have to read the referenced issue/PR and the sub references.
In a nutshell, shipped <> part of the tar source. By chance possible but not by design.
It was a misnaming and by that is in danger to misinterpretation for use cases.
The PR would have fixed that in a clean way, but it also required a change in the signing process because of various versions that are created (community, enterprise, custom ect.). It was closed due to a open PR cleanup process not because it was decided against. @PVince81 can tell you more about.

**Comment by settermjd** [comment]
@PVince81 @pmaier1 can one of you provide guidance here, please

**Comment by settermjd** [comment]
ping @PVince81.

File: modules/administration_manual/examples/installation/apps_management_installation/move-non-core-shipped-apps-to-custom-dir.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+APPS_LIST=$(sudo -u www-data ./occ app:list --shipped=false --output=json)
```
Comment (PVince81) [inline modules/administration_manual/examples/installation/apps_management_installation/move-non-core-shipped-apps-to-custom-dir.sh:3]: not all distros use www-data as user, is this script intended for a specific distro ?

Comment (mmattel) [reply modules/administration_manual/examples/installation/apps_management_installation/move-non-core-shipped-apps-to-custom-dir.sh:3]: yes, it was decided to focus in docs on Ubuntu

**Comment by mmattel** [comment]
@PVince81 please see all my comments above starting with https://github.com/owncloud/docs/pull/582#issuecomment-461049281
and the core PR with it´s discussion https://github.com/owncloud/core/pull/36389 (get_apps_in_tar)
Without the core PR being merged, this PR and all others based on that idea can´t succeed.

**Pull request closed without merging**